setup.py

```diff
@@ -50,16 +50,8 @@
 # Add the project-global data
 package_info['package_data'].setdefault('astropy', []).append('data/*')
 
-# Currently the only entry points installed by Astropy are hooks to
-# zest.releaser for doing Astropy's releases
+# Add any necessary entry points
 entry_points = {}
-for hook in [('prereleaser', 'middle'), ('releaser', 'middle'),
```

Comment:
The line above this one could probably now be merged into the ones below, since we are now only using the 'console_scripts' entry point, but no big deal either way.